src/flags.go

```diff
@@ -174,6 +175,12 @@ func propagateFlags() {
 		PanicWithMessage("Non-empty data-body was provided, but no request type was given. Hence, encoding of data-body is not possible.")
 	}
 
+	if strings.HasPrefix(strings.TrimSpace(CurrentConfig.DataText), "@") {
+		file, err := os.ReadFile(CurrentConfig.DataText[1:])
```

Comment:
add verbosity statement, that this file is being read as input

Reply:
Added